src/components/button/Button.module.scss

```diff
@@ -0,0 +1,374 @@
+@use 'theme';
+@use 'sass:meta';
+
```

Comment:
I see you use the `sass:meta`, and not seeing you use any `meta` mixin function.  Maybe it's not necessary here 🤔